src/Controller/ItemsController.ts

```diff
@@ -92,4 +94,34 @@ export class ItemsController extends BaseHttpController {
 
     return this.ok()
   }
+
+  @httpDelete('/email-backups/:userUuid')
+  public async disableEmailBackups(request: Request): Promise<results.NotFoundResult | results.OkResult> {
```

Comment:
can we put this into the `AdminController` - would make sense to group that there since this is something that is not publicly available

Reply:
Done 👍🏽